src/Symfony/Component/Serializer/Normalizer/AbstractObjectNormalizer.php

```diff
@@ -60,6 +61,7 @@ public function __construct(ClassMetadataFactoryInterface $classMetadataFactory
             $classDiscriminatorResolver = new ClassDiscriminatorFromClassMetadata($classMetadataFactory);
         }
         $this->classDiscriminatorResolver = $classDiscriminatorResolver;
+        $this->objectClassResolver = $objectClassResolver;
```

Comment:
I know it's late, but how about `$this->objectClassResolver = $objectClassResolver >: '\get_class';` or something like that ? So that we don't need to test if it is not null on each call

Reply:
@Taluu : You should provide a PR :)